modules/troubleshooting-network-observability-controller-manager-pod-out-of-memory.adoc

```diff
@@ -6,26 +6,41 @@
 [id="controller-manager-pod-runs-out-of-memory_{context}"]
 = Network Observability controller manager pod runs out of memory
 
-You can increase memory limits for the Network Observability operator by patching the Cluster Service Version (CSV), where Network Observability controller manager pod runs out of memory.
+You can increase memory limits for the Network Observability operator by editing the `spec.config.resources.limits.memory` configuration in the `Subscription` object.
 
 .Procedure
 
-. Run the following command to patch the CSV:
+. In the web console, navigate to *Operators* -> *Installed Operators*
+. Click *Network Observability* then select *Subscription*.
+. From the *Actions* menu, choose *Edit Subscription*.
+.. Alternatively, you can use the CLI to open the YAML configuration for the `Subscription` object by running the following command:
 +
 [source,terminal]
 ----
-$ oc -n netobserv patch csv network-observability-operator.v1.0.0 --type='json' -p='[{"op": "replace", "path":"/spec/install/spec/deployments/0/spec/template/spec/containers/0/resources/limits/memory", value: "1Gi"}]'
+$ oc edit subscription netobserv-operator -n openshift-netobserv-operator
 ----
+. Edit the  `Subscription` object to add the `config.resources.limits.memory` and set the memory to account for your needs:
```

Comment:
Extra space before ``Subscription` object`? Maybe it's just the PR. 

How about:

> Edit the `Subscription` object by specifying a value that meets your storage needs for  `config.resources.limits.memory`.

Reply:
The `config.resources.limits.memory` isn't in the Object by default, so the user will need to add this specification and set the memry limits..I can see adding "value" though. And yep that's an extra space, mhmm.